packages/ddp-rate-limiter/ddp-rate-limiter-tests.js

```diff
@@ -77,6 +77,66 @@ testAsyncMulti("ddp rate limiter - matchers get passed correct arguments", [
   }
 ]);
 
+testAsyncMulti("ddp rate limiter - callbacks get passed correct arguments", [
+  function (test, expect) {
+    _.extend(this, createTestUser(test, expect));
+  },
+  function (test, expect) {
+    var self = this;
+    Meteor.call("addRuleToDDPRateLimiter", expect(function(error, result) {
+     self.ruleId = result;
+    }));
+  },
+  function (test, expect) {
+    Meteor.call('dummyMethod', function() {});
+  },
+  function (test, expect) {
+    var self = this;
+    Meteor.call(
+      "getLastRateLimitEvent", expect(function (error, result) {
+        test.isTrue(result.reply.allowed);
+        test.isTrue(result.reply.timeToReset < RATE_LIMIT_INTERVAL_TIME_MS);
+        test.equal(result.reply.numInvocationsLeft, 4);
+
+        test.equal(result.ruleInput.userId, Meteor.userId());
+        test.equal(result.ruleInput.type, 'method');
+        test.equal(result.ruleInput.name, 'dummyMethod');
+      }));
+  },
+  function (test, expect) {
+    Meteor.setTimeout(expect(), RATE_LIMIT_INTERVAL_TIME_MS);
+  },
+  function (test, expect) {
+    for (var i = 0; i < RATE_LIMIT_NUM_CALLS + 1; i++) {
```

Comment:
It is to make sure the callback is triggered with the reject output (method will not be executed) of the rule.